app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -260,11 +260,11 @@ def display_phone_to_deliver_to
 
   def reenter_phone_number_path
```

Comment:
I like it! This one was actually super confusing when doing this PR, thought it was a named path and was looking all over `routes.rb` and then finally found it in the controller concern. 
updated at b124012b